<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mywindow</class>
 <widget class="QMainWindow" name="mywindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>551</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>키워드수집기</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>419</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>수집 시작</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>419</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>수집 끝내기</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio1">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>369</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>이메일 형식(메일 발송용)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio2">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>389</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>아이디 형식(쪽지 발송용)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>108</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>344</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2. 원하시는 저장 방식을 지정해주세요.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>7</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>프로그램을 이용해주셔서 감사합니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>24</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP 일시차단이 발생할 수 있으므로 수집 후 10분은 휴식해주세요.</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>246</y>
      <width>412</width>
      <height>230</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>돋움</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>중간에 종료하셔도 그때까지 수집한 이메일이 저장됩니다.</string>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>246</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>23</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1. 수집할 최종 키워드를 선택해주세요.</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>475</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>블로그/카페 이메일수집기 구매문의</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>476</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>무단복제/재배포 금지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>138</y>
      <width>65</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>55</y>
      <width>261</width>
      <height>177</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>30</y>
       <width>173</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>원하시는 검색어를 입력하세요.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>114</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>* 검색: 연관 검색어 검색</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>132</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>* 선택: 이 키워드로 진행</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>591</x>
      <y>490</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>wowon7975@naver.com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>299</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>138</y>
      <width>65</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>선택</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>73</y>
      <width>161</width>
      <height>119</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0_1">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>198</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>1차 연관검색어</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1_1">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>198</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>선택</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget3">
    <property name="geometry">
     <rect>
      <x>561</x>
      <y>73</y>
      <width>161</width>
      <height>119</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>198</y>
      <width>163</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>선택</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>541</x>
      <y>50</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>2차 연관검색어</string>
    </property>
   </widget>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>pushButton2</zorder>
   <zorder>pushButton3</zorder>
   <zorder>radio1</zorder>
   <zorder>radio2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>listWidget</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>pushButton0</zorder>
   <zorder>label_12</zorder>
   <zorder>lineEdit2</zorder>
   <zorder>pushButton1</zorder>
   <zorder>listWidget2</zorder>
   <zorder>pushButton0_1</zorder>
   <zorder>pushButton1_1</zorder>
   <zorder>listWidget3</zorder>
   <zorder>pushButton1_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
